src/main/menu/menu.ts

```diff
@@ -302,14 +307,17 @@ export function buildMenu(windowManager: WindowManager) {
   ]);
 
   // Modify menu from extensions-api
-  for (const { parentId, ...menuItem } of MenuRegistry.getInstance().getItems()) {
-    if (!appMenu.has(parentId)) {
-      logger.error(`[MENU]: cannot register menu item for parentId=${parentId}, parent item doesn't exist`, { menuItem });
+  for (const menuItem of electronMenuItems) {
```

Comment:
IMO it is cleaner to not pass `menuItem.parentId` to the list of submenus.

Reply:
Hmm. I suppose I considered the complication in typing not worth this. Do you agree? I see your point of encapsulation, but this case is trivial, and therefore simplicity is more pragmatic.

Reply:
How is the new code more simple?

Reply:
I don't see this issue as good time investment. I can change it back if you feel strongly about it :)